src/components/Grid/GridRow.tsx

```diff
@@ -20,7 +20,7 @@ export const GridRow = styled('div')`
   @media ${DESKTOP_MEDIA_QUERY} {
     grid-template-columns: repeat(12, minmax(53px, 1fr));
     grid-column-gap: 24px;
-    max-width: 1152px;
+    max-width: 85vw;
```

Comment:
how come u changed this? ur not supposd to change anything in the Grid component